javascript-modal/main.js

```diff
@@ -0,0 +1,14 @@
+const $clickButton = document.querySelector('.click-button');
+const $closeButton = document.querySelector('.closeButton');
+const popup = document.querySelector('.popup');
+
+function clickButton() {
+  popup.style.display = 'block';
+}
+
+function closeButton() {
+  popup.style.display = 'none';
+}
```

Comment:
Works, but we want to avoid using inline styling. I would add / remove a class instead